pkgs/tools/misc/codebraid/default.nix

```diff
@@ -2,16 +2,16 @@
 
 python3Packages.buildPythonApplication rec {
   pname = "codebraid";
-  version = "0.5.0-unstable-2020-08-14";
+  version = "0.11.0";
 
   src = fetchFromGitHub {
     owner = "gpoore";
     repo = pname;
-    rev = "526a223c4fc32c37d6c5c9133524dfa0e1811ca4";
-    sha256 = "0qkqaj49k584qzgx9jlsf5vlv4lq7x403s1kig8v87i0kgh55p56";
+    rev = "v${version}";
+    hash = "sha256-E9vzGK9ZEVwF+UBpSkdM+hm6vINen/A+LgnnPpc77QQ=";
   };
 
-  propagatedBuildInputs = with python3Packages; [ bespon ];
+  propagatedBuildInputs = with python3Packages; [ bespon setuptools ];
```

Comment:
`setuptools` should be in `nativeBuildInputs`

Reply:
Many thanks for those fixes. Much appreciated.